classes/classes/MainViewManager.as

```diff
@@ -295,7 +300,49 @@ package classes
 			}
 			
 		}
-		
+
+		public function hideSprite():void {
+			// Inlined from lib/src/coc/view/MainView.as
+			mainView.sprite.visible = false;
+			mainView.spriteOld.visible = false;
+		}
+		public function showSpriteFrame(frame:int):void {
+			// Inlined from lib/src/coc/view/MainView.as
+			var type:Boolean = flags[kFLAGS.SPRITE_STYLE] == 0;
+			var element:MovieClip;
+			var other:MovieClip;
+			if (type) {
+				other = mainView.spriteOld;
+				element = mainView.sprite;
+			} else if (type == 1) {
+				other = mainView.sprite;
+				element = mainView.spriteOld;
+			} else {
+				return;
+			}
+			other.visible = false;
+			element.visible = true;
+			element.gotoAndStop(frame);
+			rescaleSprite(element);
+		}
+		public function showSpriteBitmap(bmp:BitmapData):void {
```

Comment:
Going to tag my comment here for lack of a better space--

While I'm personally in favour of this new sprite call method, you've also successfully nuked 8-bit sprite support. Perhaps there should be another folder for 8-bit sprites (as to keep everything from getting messy in the sprites folder), and this function has different folder calls based on the SPRITE_STYLE flag? 0 = 8bit sprites, 1 = 16bit sprites. Obviously, nothing should be shown if sprites are disabled, but that's a whole 'nother flag and already handled :p